### DISCUSSION TITLE
Problemas al acceder a Microsoft Teams

### DISCUSSION THREAD

**Discussion by alfredoaguiararce** [LaunchX-InnovaccionVirtual/CursoIntroPython#229] [Q&A] [answered]
Buenas noches compañeros!

¿Alguien más tiene problemas al intentar acceder a Microsoft teams?

**Comment by doguedogue** [comment]
No amigo , estoy adentro sin problemas

**Comment by ismagarri** [comment]
no, sin problema. Tanto desde web como desde la app de escritorio de windows

**Comment by SebbyLebby** [comment]
Yo entré, e igual no he presentado problema. Sigues aún sin poder entrar?

**Comment by charlyguz** [comment, marked as answer]
no, yo entro normal con el acceso que mandaron por correo pero si tienes problemas por que no mandas correo a innovación contacto@innovaccion.mx
